### PR TITLE
Feature/scoreboard initial

### PR DESCRIPTION
**Pull request by davidrideout**
Scoreboard initial commit, work in progress.
Implements:
- login/logout/registration (manual registration untested)
- basic account page for editing zipcode and lobby group
- list of agenda items, agenda detail view, and a basic player for testimonies
- admin section for creating/editing agenda items

image
image
image
image
image
image
image

### CODE REVIEW

**Comment by johnkmzhou** [comment]
Could you move the firebase files out of web and to the parent directory? Also, you don't need an env variable for the firebase api key, you can just commit it in code because it's meant to be public. Before you do it though, change the firebase rules to set `allow write: if false;` to prevent anyone from writing to it until we set up authorized admins.

**Comment by davidrideout** [comment]
Files have been moved. I plan to commit firebase configs after the admin capability is done (toggling it for development is a hassle).

File: web/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { createBrowserHistory } from 'history';
```
Comment (johnkmzhou) [inline web/src/App.tsx:3]: history dependency is unnecessary. Use BrowserRouter instead.

**Comment by davidrideout** [comment]
* Swapped moment for datefns
* Added firestore and storage permissions along with a basic claims capability (stored in /permissions/{uid})
* Added image upload for agenda items

**Review by johnkmzhou — APPROVED** [review]
Looks like I forgot to submit the review I had last week with my comments but since it looks like you resolved most of the issues we've discussed, I'm going approve this one.

**Pull request merged**